### DISCUSSION TITLE
URL

### DISCUSSION THREAD

**Discussion by ghost** [tighten/jigsaw#624] [General]
There's something I wanted to ask.. when you navigation to diffrent uris for example
```
https://example.com/about/
https://example.com/contact/
```
is there a way to remove the `/` from `about/` and `contact/` so that it ends up with 
```
https://example.com/about
https://example.com/contact
```

**Comment by bakerkretzmar** [comment]
I don't think so but I'm not sure. My own site which is built with Jigsaw has links without the trailing slash but shows them in the browser, and even redirects to them if I try to edit the URL manually. It's hosted on Netlify and I'm using Firefox, my guess is this is a browser/hosting thing.

**Reply by ghost** [reply]
no i found otu why, is one of the settings with Netlify... now the trailing slash is no longer there.. 
https://support.gatsbyjs.com/hc/en-us/articles/360056349113-Remove-trailing-slash-in-Netlify-URLs
